tests/Feature/WebStoreRouteTest.php

```diff
@@ -64,4 +66,27 @@ public function test_store_import_csv_stats()
         $response->assertHeader('Content-Type', 'text/csv; charset=utf-8');
         $response->assertDownload($filename);
     }
+
+    public function test_store_import_results_route_import_id_does_not_exist()
+    {
+        $response = $this->get(route('store.import-results', ['123123123']));
+        $response->assertStatus(404);
+    }
+
+    public function test_store_import_results_route()
+    {
+        $filename = strtr(config('imports.results.filename_template'), [
+            ':datetime' => now()->format('Y-m-dTH:i:s')
+        ]);
```

Comment:
```suggestion
            ':id' => $import->id
            ':datetime' => now()->format('Y-m-dTH:i:s')
        ]);
```

Same as the above comments regarding the change in the filename template.
ofc this will also require us to determine the ImportMeta prior to the filename assignment.